storage/Devices/Partitionable.h

```diff
@@ -0,0 +1,58 @@
+#ifndef STORAGE_PARTITIONABLE_H
+#define STORAGE_PARTITIONABLE_H
+
+
+#include <vector>
+
+#include "storage/Devices/BlkDevice.h"
+#include "storage/Devices/PartitionTable.h"
+#include "storage/StorageInterface.h"
+
+
+namespace storage
+{
+
+    // abstract class
```

Comment:
It is abstract since no objects of type Partitionable can be created. Only objects of derived classes (currently Disk, Md) can be created. That is standard C++ terminology.